tests/pages/firefox/base.py

```diff
@@ -2,17 +2,25 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from pypom import Region
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
 
 from pages.base import BasePage
-from pages.page import PageRegion
 
-# this should be enough to scroll elements so they are not beneath the header
-HEADER_OFFSET = -100
```

Comment:
I can imaging this value might need to change in future if/when we update the nav. For this reason I think making it more obvious what the dev needs to change is worthwhile. I would vote to either restore it, or make it more obvious via a comment in the function itself. Either would be fine.